whois-endtoend/src/test/groovy/net/ripe/db/whois/spec/update/AsSetSpec.groovy

```diff
@@ -1541,14 +1578,16 @@ class AsSetSpec extends BaseQueryUpdateSpec {
     }
 
     def "create as-set object with all optional attrs"() {
+      given:
+        dbfixture(getTransient("AS123"))
```

Comment:
This change creates the object as a fixture already, it changes the intent of the test

Reply:
Better to use `spock.lang.Ignore` than change the intent